fabric-1.17.1/gradle.properties

```diff
@@ -0,0 +1,12 @@
+# Done to increase the memory available to gradle.
+org.gradle.jvmargs=-Xmx1G
+# Fabric Properties
+# check these on https://fabricmc.net/use
+minecraft_version=1.17.1
+yarn_mappings=1.17.1+build.1
+loader_version=0.11.6
+# Mod Properties
+archives_base_name=Dynmap
+# Dependencies
+# currently not on the main fabric site, check on the maven: https://maven.fabricmc.net/net/fabricmc/fabric-api/fabric-api
+fabric_version=0.36.1+1.17
```

Comment:
I think this needs to be fixed

Reply:
Nope, this is correct. Thanks for the callout though.

See https://maven.fabricmc.net/net/fabricmc/fabric-api/fabric-api/0.36.1%2B1.17/ and https://www.curseforge.com/minecraft/mc-mods/fabric-api/files/3377591